### PR TITLE
Link to documentation instead of embedding help

### PR DESCRIPTION
**Pull request by HenningCash**
## :loudspeaker: Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## :scroll: Description
This is a temporary solution for the help page. The current help page was not very helpful as it only included extracts of our full documentation. Additionally, we had a problem with the asciidoctor.js library messing around with global classes (`Error` in our case).

To make people aware of the full docs this will replace the "Help" link in the Sidebar and Embedded Help-Links with links to `docs.codefreak.org`. In the long run we should embed the full documentation into the application but this will take a lot of work.

### CODE REVIEW

File: client/src/components/CodefreakDocsLink.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  category?: 'for-admins' | 'for-teachers' | 'for-students' | 'for-developers'
+  page?: string
+  section?: string
+}
+
+const CodefreakDocsLink: React.FC<CodefreakDocsLinkProps> = props => {
+  const { category, page = 'index', section, children } = props
+
+  // strip trailing slashes from the base URL
+  let link = process.env.CODEFREAK_DOCS_BASE_URL.replace(/^\/+/, '')
```
Comment (erikhofer) [inline client/src/components/CodefreakDocsLink.tsx:13]: doesn't this match leading slashes? 🤔

File: client/src/components/CodefreakDocsLink.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  if (category) link += `/${category}`
+  link += `/${page}.html`
+  if (section) link += `#${section}`
+
+  return (
+    <a href={link} target="_blank" rel="noreferrer">
```
Comment (erikhofer) [inline client/src/components/CodefreakDocsLink.tsx:19]: add noopener

File: client/src/router.config.tsx [hunk 3]
```diff
@@ -56,38 +48,12 @@ export const routerConfig: Route = {
...
-      ]
-    },
-    {
-      path: '/admin',
-      name: 'Administration',
-      icon: <SettingOutlined />,
-      authority: AUTHORITIES.ROLE_ADMIN,
-      component: AdminPage
+      isUrl: true,
+      target: '_blank',
```
Comment (erikhofer) [inline client/src/router.config.tsx:55]: can we add rel="noopener noreferrer" here?

File: client/src/components/CodefreakDocsLink.tsx (hunk 1 above)
Comment (HenningCash) [reply client/src/components/CodefreakDocsLink.tsx:13]: d'oh :joy:

File: client/src/router.config.tsx (hunk 3 above)
Comment (HenningCash) [reply client/src/router.config.tsx:55]: It's possible with a custom component

**Review by erikhofer — APPROVED** [review]

**Pull request merged**